Tests/test_Prank_tool.py

```diff
@@ -26,9 +26,11 @@
     # For Windows, PRANK just comes as a zip file which contains the
     # prank.exe file which the user could put anywhere.  We'll try a few
     # sensible locations under Program Files... and then the full path.
-    likely_dirs = ["",  # Current dir
-                   prog_files,
-                   os.path.join(prog_files, "Prank")] + sys.path
+    likely_dirs = [
+        "",  # Current dir
```

Comment:
That looks like an old bug, "" does not mean current directory but "." would. If you have access to Windows would you be able to try this?

Reply:
Oh, didn't catch that. Don't have access to windows though, unfortunately.

Reply:
Nor me anymore, unfortunately. Perhaps it really does work on Windows? Let's leave it alone for now.